flink-core/src/main/java/org/apache/flink/api/connector/sink/GlobalCommitter.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.flink.api.connector.sink;
+
+import org.apache.flink.annotation.Experimental;
+
+import java.util.List;
+
+/**
+ * The {@link GlobalCommitter} is responsible for committing an aggregated committable, which we called global committable.
+ *
+ * @param <CommT>         The type of the committable data
+ * @param <GlobalCommT>   The type of the aggregated committable
+ */
+@Experimental
+public interface GlobalCommitter<CommT, GlobalCommT> extends Committer<GlobalCommT> {
+
+	/**
+	 * Find out which global committables need to be retried when recovering from the failure.
+	 * @param globalCommittables the global committable that are properly not committed in the previous attempt.
```

Comment:
What about: "A list of < GlobalCommT> for which we want to verify which ones were successfully committed and which ones did not."